slicops/ui_api.py

```diff
@@ -19,5 +20,23 @@ class UIAPI(slicops.quest.API):
     Or, raise a `pykern.quest.APIError`.
     """
 
+    def _app_implementation(self, api_args):
+        assert 'app_name' in api_args
```

Comment:
Let's create a class for this that we can convert. @e-carlin maybe share my notes with @moellep.